docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,254 @@ title: React Native SDK Migration Guide
 description: Use this guide to easily migrate from Amplitude's maintenance React Native SDK (@amplitude/react-native) to the new SDK (@amplitude/analytics-react-native).
 ---
 
+!!! warning
+    The Maintenance React Native SDK stores device ID, user ID, etc., in SQLite. However, there is no migration logic available (at this time), so this data won't be automatically migrated when upgrading to the new React Native SDK. Consequently, for example, device ID may be regenerated when using the new SDK.
+
+    We will release a migration plugin soon.
+
+Amplitude new React Native SDK (`@amplitude/analytics-react-native`) features a plugin architecture, built-in type definition and broader platform support. New React Native SDK isn't backwards compatible with maintenance React Native SDK `@amplitude/react-native`. 
+
+To migrate to `@amplitude/analytics-react-native`, update your dependencies and instrumentation.
+
+## Terminology
+
+* `@amplitude/react-native`: Maintenance React Native SDK
+* `@amplitude/analytics-react-native`: New React Native SDK
+
+## Dependency
+
+Uninstall `@amplitude/react-native` by either `yarn remove @amplitude/react-native` or deleting your dependency list in package.json
+
+```diff
+{
+    "dependencies": {
+-    "@amplitude/react-native": "^2"
```

Comment:
```suggestion
-    "@amplitude/react-native": "*"
+    "@amplitude/analytics-react-native": "^1"
```

Reply:
I added a link to new RN installation section in case of major version release

Reply:
Thanks, I saw the link but still think it would be good to show the required dependency change here as well. Ideally the customer can follow this doc top to bottom without having to leave the page for supplementary info.